### DISCUSSION TITLE
Filebrowser not reachable

### DISCUSSION THREAD

**Discussion by Piero87** [tteck/Proxmox#3457] [Help Needed]
Hello, I have installed Filebrowser in my LXC container using the command in the helper script:

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/filebrowser.sh)"

this is the result:
<img width="604" alt="Screenshot 2024-08-02 at 00 27 27" src="https://github.com/user-attachments/assets/fe395a64-273b-44e9-9ebe-5e7097a7a14d">

but if I try to open FileBrowser the browser tell me is not reachable:

<img width="1425" alt="Screenshot 2024-08-02 at 00 31 17" src="https://github.com/user-attachments/assets/0df39865-ef92-4df8-9ba3-a7779a9425d9">

what I'm missing?

**Comment by tteck** [comment]
No issue here

```
    _______ __     ____
   / ____(_) /__  / __ )_________ _      __________  _____
  / /_  / / / _ \/ __  / ___/ __ \ | /| / / ___/ _ \/ ___/
 / __/ / / /  __/ /_/ / /  / /_/ / |/ |/ (__  )  __/ /
/_/   /_/_/\___/_____/_/   \____/|__/|__/____/\___/_/

Would you like to use No Authentication? <y/N> y
 ✓ Installed FileBrowser on apt-cacher-ng
 ✓ Created Service
 ✓ Completed Successfully!

FileBrowser should be reachable by going to the following URL.
         http://192.168.86.109:8080 

root@apt-cacher-ng:~# 
```

![Screenshot 2024-08-01 6 36 19 PM](https://github.com/user-attachments/assets/93a78dae-4f50-4f59-8bb3-7560d67200de)

**Reply by Piero87** [reply]
Thanks for your answer, I have installed it inside my realdebrid client lxc used by your script, can you try there?

**Reply by Piero87** [reply]
@tteck I have tried to install File Browser inside this LXC:
<img width="1149" alt="Screenshot 2024-08-02 at 08 47 29" src="https://github.com/user-attachments/assets/1822509a-3526-406b-94ab-e3e5ba9994a6">
and it's working, instead if I install inside this LXC:
<img width="1147" alt="Screenshot 2024-08-02 at 08 47 41" src="https://github.com/user-attachments/assets/62f11c02-d298-411c-b986-15452aafd085">
it's not reachable, is it possible to put it inside the real debrid lxc too?
